src/app/utils/settings.ts

```diff
@@ -0,0 +1,19 @@
+import Cookies from "js-cookie"
+
+import { SERVICE_NAME } from "codeforlife/settings"
+import { type SettingsState } from "../slices"
+
+export function getSettingCookie() {
```

Comment:
spelling - be careful please - it's "~setting~" --> "settings"
```suggestion
export function getSettingsCookie() {
```